test/unittest/ext/llm/utils.py

```diff
@@ -0,0 +1,73 @@
+"""
+LLM model test cases.
+All the llm model can use the check_xxx func to test the intergration with db.
+"""
+
+from superduperdb.backends.ibis.field_types import dtype
+from superduperdb.backends.ibis.query import Schema, Table
+from superduperdb.backends.mongodb.data_backend import MongoDataBackend
+from superduperdb.backends.mongodb.query import Collection
+from superduperdb.base.document import Document
+from superduperdb.components.listener import Listener
+
+
+def check_predict(db, llm):
+    """Test whether db can call model prediction normally."""
+    db.add(llm)
+    result = db.predict(llm.identifier, "1+1=")[0].content
+    assert isinstance(result, str)
+
+    results = db.predict(llm.identifier, ["1+1=", "2+2="])[0].content
+
+    assert isinstance(results, list)
+    assert len(results) == 2
+
+
+def check_llm_as_listener_model(db, llm):
```

Comment:
is this more of test suite utils method?

Reply:
Yes, we need to move this to the test suite utils tool in the future